crates/relayer/src/chain/penumbra/config.rs

```diff
@@ -0,0 +1,75 @@
+use core::time::Duration;
+
+use ibc_relayer_types::core::ics24_host::identifier::ChainId;
+use penumbra_custody::soft_kms;
+use serde_derive::{Deserialize, Serialize};
+use tendermint_rpc::Url;
+
+use crate::config::{default, types::TrustThreshold, EventSourceMode, PacketFilter, RefreshRate};
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct PenumbraConfig {
+    /// A fake key name, not used except to satisfy external interfaces.
+    ///
+    /// The Config::key_name() method returns &String, forcing configs to own a String.
+    /// We don't want to just change the method, because we want our fork to apply cleanly
+    /// onto upstream, so we need to
+    /// 1. use a stub field (here)
+    /// 2. create a separate PR to change the method signature
+    /// 3. rebase our change stack once it's merged and remove the stub field.
+    #[serde(default)]
+    pub stub_key_name: String,
+
+    pub id: ChainId,
+    /// The URL of the app's gRPC endpoint.
+    pub grpc_addr: Url,
+    /// The URL of the cometbft RPC endpoint.
+    ///
+    /// TODO: in the long term, it would be desirable to eliminate this entirely.
+    pub rpc_addr: Url,
```

Comment:
@avahowell @conorsch branching off of discord discussion https://discord.com/channels/824484045370818580/930154881040404480/1202338530630840392

The existing cosmos endpoint code talks to both the sdk app and to comet, it's likely that we'll need that as well, at least in the short term. Longer term it would be preferable to avoid any use of the comet rpc endpoint, but this doesn't seem critical to put a lot of effort into in the near term.